ecommerce-demo/workspaces/childDemo_bigboxretailer/workspace.json

```diff
@@ -0,0 +1,17 @@
+{
+  "data": {
+    "attributes": {
+      "name": "E-commerce Merchant Bigboxretailer"
+    },
+    "id": "ecommerce-merchant-bigboxretailer",
```

Comment:
Should we add `demo_` as a prefix for the id? Also should we change hyphens `-` to underscores `_`, to be consistent with the naming of other ids (e.g., the colorPaletteId and themeId)?

Something like:

```json
"id": "demo_ecommerce_merchant_bigboxretailer",
```